static/templates/intro_reply_hotspot.hbs

```diff
@@ -1,4 +1,4 @@
-<div class="hotspot-inline trailing_bookend bookend hotspot-message" id="hotspot_intro_reply_icon">
+<div class="hotspot-inline trailing_bookend bookend hotspot-message" id="hotspot_intro_reply_overlay">
```

Comment:
If we decide to remove this commit, I'll have to add custom logic in the `hotspots.is_hotspot_exist` for intro reply hotspot.